acceptance/tests/pxp-module-puppet/run_puppet_result_changed.rb

```diff
@@ -0,0 +1,65 @@
+require 'pxp-agent/test_helper.rb'
+
+ENVIRONMENT_NAME = 'changed_result'
+MODULE_NAME = 'changed'
+
+test_name 'C93062 - Run puppet and expect \'changed\' result' do
+
+  step 'On master, create a new environment that will result in Changed' do
+    environmentpath = master.puppet['environmentpath']
+    site_manifest = "#{environmentpath}/#{ENVIRONMENT_NAME}/manifests/site.pp"
+    on(master, "cp -r #{environmentpath}/production #{environmentpath}/#{ENVIRONMENT_NAME}")
+    create_remote_file(master, site_manifest, <<-SITEPP)
+node default {
+  include #{MODULE_NAME}
```

Comment:
This use of a module seems overkill.  The entire site.pp could be just the notify resource.